llvm/CMakeLists.txt

```diff
@@ -517,7 +517,9 @@ option(LLVM_ENABLE_LIBEDIT "Use libedit if available." ON)
 option(LLVM_ENABLE_LIBPFM "Use libpfm for performance counters if available." ON)
 
 # On z/OS, threads cannot be used because TLS is not supported.
-if (CMAKE_SYSTEM_NAME MATCHES "OS390")
+# EraVM local begin
```

Comment:
Better is to put TODO with ticket number here as we review local changes from time to time.

Reply:
Do you meant TODO along with "EraVM local begin", or just TODO?